### DISCUSSION TITLE
Buttons don't look right with CDN

### DISCUSSION THREAD

**Discussion by bfg-coding** [saadeghi/daisyui#2732] [Q&A]
Has anyone had any issue with their buttons not coloring right when using the cdn?

![Screenshot from 2024-01-04 21-22-10](https://github.com/saadeghi/daisyui/assets/11151592/fb543e27-a7f3-4f18-b2c1-31cfa474af62)

**Comment by saadeghi** [comment]
Can you share the HTML code for the button on this screenshot?

**Comment by bfg-coding** [comment]
Can't show the whole html but here's the parts I think would provide context

```html
<!DOCTYPE html>
<html data-theme="dark" lang="en">

<head>
  <meta charset="UTF-8">
  <title>{% block title %}Document{% endblock %}</title>

  <link href="https://cdn.jsdelivr.net/npm/daisyui@4.4.24/dist/full.min.css" rel="stylesheet" type="text/css" />
  <script src="https://cdn.tailwindcss.com?plugins=typography"></script>
  <script src="https://unpkg.com/htmx.org@1.9.10"
    integrity="sha384-D1Kt99CQMDuVetoL1lrYwg5t+9QdHe7NLX/SoJYkXDFfX37iInKRy5xLSi8nO7UC"
    crossorigin="anonymous"></script>
</head>

```

```html
  <div class="flex flex-row gap-3">
      <button type="submit" class="btn btn-primary">
          Update
      </button>
  </div>
```

**Comment by stijnvgeene** [comment]
I have the same issue and haven't found a solution yet. I did find that it's caused by button type="submit".

**Comment by philharmonie** [comment]
This is related to `type=submit`. Remove it and your buttons look right
